lib/WP_Auth0_DBManager.php

```diff
@@ -164,15 +158,6 @@ public function install_db( $version_to_install = null, $app_token = '' ) {
 
 				$payload = array(
 					'app_type' => 'regular_web',
-					'callbacks' => array(
```

Comment:
This was overwriting existing Application settings, which could have unintended effects.